aspnetcore/host-and-deploy/iis/troubleshoot.md

```diff
@@ -81,6 +81,86 @@ The ASP.NET Core Module fails to find the out-of-process hosting request handler
 
 ::: moniker-end
 
+::: moniker range="= aspnetcore-3.0"
+
+### 500.30 In-Process Startup Failure
+
+The worker process fails. The app doesn't start.
+
+The ASP.NET Core Module attempts to start the .NET Core CLR in-process, but it fails to start. The cause of a process startup failure can usually be determined from entries in the [Application Event Log](#application-event-log) and the [ASP.NET Core Module stdout log](#aspnet-core-module-stdout-log).
+
+### 500.31 ANCM Failed to Find Native Dependencies
+
+The worker process fails. The app doesn't start.
+
+The ASP.NET Core Module attempts to start the .NET Core CLR in-process, but it fails to start. The cause of this startup failure most likely due to the version of `Microsoft.NETCore.App` or `Microsoft.AspNetCore.App` not being found. For example, if the application is deployed to target 3.0 and that version doesn't exist on the machine, this error can occur. The specific error will look like the following:
+
+```
+The specified framework 'Microsoft.NETCore.App', version '3.0.0' was not found.
+  - The following frameworks were found:
+      2.2.1 at [C:\Program Files\dotnet\x64\shared\Microsoft.NETCore.App]
+      3.0.0-preview5-27626-15 at [C:\Program Files\dotnet\x64\shared\Microsoft.NETCore.App]
+      3.0.0-preview6-27713-13 at [C:\Program Files\dotnet\x64\shared\Microsoft.NETCore.App]
+      3.0.0-preview6-27714-15 at [C:\Program Files\dotnet\x64\shared\Microsoft.NETCore.App]
+      3.0.0-preview6-27723-08 at [C:\Program Files\dotnet\x64\shared\Microsoft.NETCore.App]
+```
+
+To fix this error, either:
+* Install the appropriate version of dotnet on the machine.
+* Target the correct version of dotnet.
+* Publish the application as Standalone.
+
+When running in development (ASPNETCORE_ENVIRONMENT is set to Development), the specific error that occured will be written to the HTTP response. The cause of a process startup failure can also be found in the [Application Event Log](#application-event-log).
+
+### 500.32 ANCM Failed to Load dll
+
+The worker process fails. The app doesn't start.
+
+The application was likely published for a different bitness than w3wp.exe/iisexpress.exe is running as. For example, if the worker process is running as a 32-bit application and the application was published to target 64-bit, this error will occur. The first dll that will hit this error will be `hostfxr.dll`, which is responsible for starting the dotnet application.
```

Comment:
```suggestion
The most common cause for this is that the app was published for an incompatible processor architecture. For example, if the worker process is running as a 32-bit application and the application was published to target 64-bit, this error will occur.
```

Re `The first dll that will hit this error will be `hostfxr.dll`, which is responsible for starting the dotnet application.`. Is that relevant to the user? Do they see that name in the error anywhere?